ocaml/xapi/xapi_vdi_helpers.ml

```diff
@@ -23,6 +23,19 @@ open Threadext
 module D=Debug.Make(struct let name="xapi" end)
 open D
 
+let attach_rrd_vdi ~__context ~sr=
```

Comment:
Is this function going to be changed to attach the VDI later? At the moment it only creates a VDI, so the function name is a bit misleading.